docs/api/nodejs-sdk.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 3
+---
+
+# Node.js SDK API Reference
+
+Complete technical API reference for the Saiki Node.js SDK. For guides and examples, see the [SDK Guide](/docs/guides/nodejs-sdk).
+
+## Installation
+
+```bash
+npm install @truffle-ai/saiki
+```
+
+## Core Imports
+
+```typescript
+import {
+  createSaikiAgent,
+  SaikiAgent,
+  MCPManager,
+  Logger,
+  AgentEventBus,
+  SessionEventBus,
+  createStorageBackends,
+  createAgentServices
+} from '@truffle-ai/saiki';
+```
+
+## API Sections
+
+### [SaikiAgent Class](./saiki-agent)
+Complete API reference for the main `SaikiAgent` class:
+- Factory function (`createSaikiAgent`)
+- Core methods (`run`, session management)
+- Configuration management
+- MCP server integration
+
+### [Standalone Utilities](./utilities)
+Individual classes for modular usage:
+- `MCPManager` - MCP server management
+- `Logger` - Configurable logging
+- `AgentEventBus` & `SessionEventBus` - Event handling
+- Storage and service factories
```

Comment:
only MCPManager is useful as a standalone utility

logger and event bus are just lightweight wrappers on in-built primitives so nobody is going to use them

storage also is agent specific and not standalone